contents.txt

```diff
@@ -3112,234 +2511,66 @@
-./sdk/6.0.100/TestHost/testhost.net47.x86.exe
-./sdk/6.0.100/TestHost/testhost.net47.x86.exe.config
-./sdk/6.0.100/TestHost/testhost.net48.exe
-./sdk/6.0.100/TestHost/testhost.net48.exe.config
-./sdk/6.0.100/TestHost/testhost.net48.x86.exe
-./sdk/6.0.100/TestHost/testhost.net48.x86.exe.config
-./sdk/6.0.100/TestHost/testhost.x86.exe
-./sdk/6.0.100/TestHost/testhost.x86.exe.config
-./sdk/6.0.100/TestHost/tr/
-./sdk/6.0.100/TestHost/tr/Microsoft.TestPlatform.CommunicationUtilities.resources.dll
-./sdk/6.0.100/TestHost/tr/Microsoft.TestPlatform.CoreUtilities.resources.dll
-./sdk/6.0.100/TestHost/tr/Microsoft.TestPlatform.CrossPlatEngine.resources.dll
-./sdk/6.0.100/TestHost/tr/Microsoft.TestPlatform.Utilities.resources.dll
-./sdk/6.0.100/TestHost/tr/Microsoft.VisualStudio.TestPlatform.Common.resources.dll
-./sdk/6.0.100/TestHost/tr/Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll
-./sdk/6.0.100/TestHost/x64/
-./sdk/6.0.100/TestHost/x64/msdia140.dll
-./sdk/6.0.100/TestHost/x64/msdia140.dll.manifest
-./sdk/6.0.100/TestHost/x86/
-./sdk/6.0.100/TestHost/x86/msdia140.dll
-./sdk/6.0.100/TestHost/x86/msdia140.dll.manifest
-./sdk/6.0.100/TestHost/zh-Hans/
-./sdk/6.0.100/TestHost/zh-Hans/Microsoft.TestPlatform.CommunicationUtilities.resources.dll
-./sdk/6.0.100/TestHost/zh-Hans/Microsoft.TestPlatform.CoreUtilities.resources.dll
-./sdk/6.0.100/TestHost/zh-Hans/Microsoft.TestPlatform.CrossPlatEngine.resources.dll
-./sdk/6.0.100/TestHost/zh-Hans/Microsoft.TestPlatform.Utilities.resources.dll
-./sdk/6.0.100/TestHost/zh-Hans/Microsoft.VisualStudio.TestPlatform.Common.resources.dll
-./sdk/6.0.100/TestHost/zh-Hans/Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll
-./sdk/6.0.100/TestHost/zh-Hant/
-./sdk/6.0.100/TestHost/zh-Hant/Microsoft.TestPlatform.CommunicationUtilities.resources.dll
-./sdk/6.0.100/TestHost/zh-Hant/Microsoft.TestPlatform.CoreUtilities.resources.dll
-./sdk/6.0.100/TestHost/zh-Hant/Microsoft.TestPlatform.CrossPlatEngine.resources.dll
-./sdk/6.0.100/TestHost/zh-Hant/Microsoft.TestPlatform.Utilities.resources.dll
-./sdk/6.0.100/TestHost/zh-Hant/Microsoft.VisualStudio.TestPlatform.Common.resources.dll
-./sdk/6.0.100/TestHost/zh-Hant/Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll
+./sdk/6.0.100/TestHost/ref/
+./sdk/6.0.100/TestHost/ref/testhost.dll
+./sdk/6.0.100/TestHost/ref/testhost.x86.dll
+./sdk/6.0.100/testhost.runtimeconfig.json
+./sdk/6.0.100/TestHost/testhost.deps.json
+./sdk/6.0.100/TestHost/testhost.dll
+./sdk/6.0.100/TestHost/testhost.dll.config
+./sdk/6.0.100/TestHost/testhost.runtimeconfig.json
+./sdk/6.0.100/TestHost/testhost.x86
+./sdk/6.0.100/TestHost/testhost.x86.deps.json
+./sdk/6.0.100/TestHost/testhost.x86.dll
+./sdk/6.0.100/TestHost/testhost.x86.dll.config
+./sdk/6.0.100/TestHost/testhost.x86.runtimeconfig.json
+./sdk/6.0.100/testhost.x86
+./sdk/6.0.100/testhost.x86.deps.json
+./sdk/6.0.100/testhost.x86.dll
+./sdk/6.0.100/testhost.x86.dll.config
+./sdk/6.0.100/testhost.x86.runtimeconfig.json
```

Comment:
> [@omajid]
> This one looks somewhat interesting:
> 
> ```diff
> +./sdk/6.0.100/testhost.deps.json
> +./sdk/6.0.100/testhost.dll
> +./sdk/6.0.100/testhost.dll.config
> +./sdk/6.0.100/testhost.runtimeconfig.json
> +./sdk/6.0.100/testhost.x86
> +./sdk/6.0.100/testhost.x86.deps.json
> +./sdk/6.0.100/testhost.x86.dll
> +./sdk/6.0.100/testhost.x86.dll.config
> +./sdk/6.0.100/testhost.x86.runtimeconfig.json
> ```
> 
> The name is not what I expected: x86 instead of x64? In RC2,
> testhost.x86 was also a native binary that doesn't have debug info, so I am not even sure where that was coming from. That it's also missing in the Microsoft-built SDK is very strange. Is that something you folks are investigating?